src/form-builder/CheckboxFieldGroup.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {
+  useField,
+  FieldHookConfig,
+  FieldArray,
+  useFormikContext,
+} from 'formik';
+
+import {
+  chainArrayValidations,
+  required,
+  requiredLength,
+} from '../utils/validation';
+import CheckboxField from './CheckboxField';
+import { CheckboxGroupProps, CheckboxProps } from './types';
+
+// import CheckboxGroup from 'local-web-components/CheckboxGroup';
+import CheckboxGroup from '@department-of-veterans-affairs/component-library/CheckboxGroup';
```

Comment:
There's a web component for this that should probably be used instead. The React `<CheckboxGroup>` will likely be removed in the future.

Reply:
Are there react bindings for `va-checkbox-group`? I was probably looking in the wrong place, but couldn't find any.

Reply:
Yeah, there are. They are generated automatically for each web component so they aren't listed anywhere, but you can see them in the dependency's exports in `vets-website`:

![image](https://user-images.githubusercontent.com/2008881/163589063-da22a37d-add1-4649-86af-81f9037ce69c.png)

You'll also need to be on [version 4.2.1](https://github.com/department-of-veterans-affairs/component-library/releases/tag/v4.2.1) or higher.

Reply:
Ah that explains it. Outdated version. @mlauren Feel free to include a version update in our `package.json` in your PR. As @bkjohnson mentioned, you'll need a higher version to get `VaCheckboxGroup`.

Reply:
@micahchiang @bkjohnson added the checkbox group in and updated web components version. Thanks!